### PR TITLE
new node: LOADER to send measurements to frontier

### PR DESCRIPTION
**Pull request by smahmed776**
### Description
This node sends all the input measurements to the Frontier measurements API.

### Parameters
This node takes only one parameter, that is `measurement_uuid`

### Inputs & Outputs
It takes any node as its input and returns the output of that node in the `DataContainer` class.

### Example APPS
https://github.com/flojoy-io/apps/pull/2

### Integrating the Node
- Click on `Add Node` from the top left corner.
- From node categories expand `Loaders`.
- Expand `Cloud_DATABASE` and you'll find the `LOADER` node.
- Click on it to add it to the flow chart.

This is a corresponding PR of this: https://github.com/flojoy-io/studio/pull/294 in the studio repository.

### CODE REVIEW

File: LOADERS/LOADER.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import os
+import requests
+import json
+from pathlib import Path
+import yaml
+from datetime import datetime
+
+FRONTIER_HOST= os.environ.get('FRONTIER_HOST')
+FORNTIER_PORT = os.environ.get('FRONTIER_PORT')
+API_URI= f"https://{FRONTIER_HOST}:{FORNTIER_PORT}/measurements"
```
Comment (jackparmer) [inline LOADERS/LOADER.py:12]: spelling error here? `FORNTIER_PORT` ?

Comment (smahmed776) [reply LOADERS/LOADER.py:12]: Fixed this typo. Thanks.

File: LOADERS/LOADER.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    path = os.path.join(home, '.flojoy/credentials')
+    if os.path.exists(path):
+        stream = open(path, 'r')
+        yaml_dict = yaml.load(stream, Loader=yaml.FullLoader)
+        if isinstance(yaml_dict, str) == True:
+            split_by_line = yaml_dict.split('\n')
+            for line in split_by_line:
+                if 'FRONTIER_API_KEY' in line:
+                    api_key = line.split(':')[1]
+        else:
```
Comment (jackparmer) [inline LOADERS/LOADER.py:40]: how do we let the user know if no API key is found?

Comment (jackparmer) [reply LOADERS/LOADER.py:40]: might want to add a check in `flojoy.sh` also and alert the user that their API key has not been set

each of these error and warning messages should have a link to the frontier page where the user sets their api key

Comment (smahmed776) [reply LOADERS/LOADER.py]: I have added check functionality in `flojoy` script file for the Frontier API key and also updated `LOADER` node to raise an error if any of these two keys are not found.

**Pull request merged**